### PR TITLE
Add e2e test to be executed per each new PR

### PR DESCRIPTION
**Pull request by machacekondra**
This PR add e2e test to be executed per each PR on travis on Minikube

### CODE REVIEW

**Comment by machacekondra** [comment]
@pkliczewski @mmazur @vatsalparekh

File: .travis.yml [hunk 1]
```diff
@@ -1,23 +1,38 @@
...
-  - KUBECONFIG=tests/k8s/config
-install:
-  - pip install -r requirements-dev.txt
-script:
-  - pytest --cov=lib tests
-  - flake8 --version
-  - flake8 lib/ansible/modules
+sudo: required
+dist: xenial
+
```
Comment (pkliczewski) [inline .travis.yml]: Which version of python we will be using? I think we need to run py2 and py3.

Comment (mmazur) [reply .travis.yml]: pytest from python3 will fail, so that'll warrant a separate PR to get all of that working

Comment (machacekondra) [reply .travis.yml]: Ah, I accidentally removed uni tests, will re-add.

But more imporatant is to actually try to run both Ansible with Py3 and with Py2 I guess, I make a note and will add it. On f29 Py3 is default, not sure about Ubuntu, will check.

Comment (mmazur) [reply .travis.yml]: I've checked it before and it complains, so adding py3 testing will require figuring out what's wrong. I suggest limiting this PR to what it says in this PR's title and then doing a separate 'add python3 compatibility and tests' push.

Comment (pkliczewski) [reply .travis.yml]: Ack, will open an issue to enable py3

**Pull request merged**